ArraySorting.cls

```diff
@@ -0,0 +1,53 @@
+/*3. Method which will accept array of integers as parameters write a code to
+a. Sort the array in ascending order
+b. Sort the array in descending order
+c. Print the Maximum number in the array
+d. Print the Minimum number in the array
+e. Print duplicates and its occurrences in the array*/
+
+//a. Sort the array in ascending order
+public class ArraySorting{
+	public static void arrayOrder(){

```

Comment:
@pranalimm Where is Input parameter? Read question carefully `Method which will accept array of integers as parameters`

- [ ] update code as per question.